### PR TITLE
[fix] file not found error for env binary

### PR DESCRIPTION
**Pull request by antuspenskiy**
Hello!

Fixed small problem with env path `FileNotFoundError: [Errno 2] No such file or directory: '/bin/env'`, when trying to install when my env in another path.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # importing required module
 # -------------------------------
 import subprocess
+import shutil
```
Comment (shaeinst) [inline setup.py:5]:
```suggestion
from shutil import which
```

Comment (shaeinst) [reply setup.py:5]: since there only one use case for shutil, just import what we need

File: setup.py [hunk 2]
```diff
@@ -59,7 +60,8 @@
 
 # directory locations
 # -------------------------------
-nvim = attach('child', argv=["/bin/env", "nvim", "--embed", "--headless"])
+env_binary = shutil.which("env")
```
Comment (shaeinst) [inline setup.py:63]:
```suggestion

```

File: setup.py [hunk 3]
```diff
@@ -59,7 +60,8 @@
...
+nvim = attach('child', argv=[env_binary, "nvim", "--embed", "--headless"])
```
Comment (shaeinst) [inline setup.py:64]:
```suggestion
nvim = attach('child', argv=[which("env"), "nvim", "--embed", "--headless"])
```

File: setup.py (hunk 1 above)
Comment (AlixShahid) [inline setup.py]:
```suggestion
from shutil import which
```

File: setup.py (hunk 2 above)
Comment (AlixShahid) [inline setup.py]:
```suggestion

```

File: setup.py (hunk 3 above)
Comment (AlixShahid) [inline setup.py]:
```suggestion
nvim = attach('child', argv=[which("env"), "nvim", "--embed", "--headless"])
```

**Review by AlixShahid — COMMENTED** [review]
since there is only one use case for shutil, just import what we need

**Comment by antuspenskiy** [comment]
> since there is only one use case for shutil, just import what we need

looks good to me.

**Comment by AlixShahid** [comment]
thanks for PR

**Pull request merged**